virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -74,7 +77,10 @@ If you disable dynamic key injection, the VM inherits the key management method
 :dynamic-key:
 include::modules/virt-adding-key-creating-vm-template.adoc[leveloffset=+3]
 
+// Hiding in ROSA/Dedicated as this is a Dev Preview feature
+ifndef::openshift-rosa,openshift-dedicated[]
 include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+3]
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
Ditto re: instance types support. If instance types are supported in 4.15+, should this be hidden?